tests/runner.d

```diff
@@ -186,6 +218,12 @@ int main(string[] args)
         return regressed ? -1 : 0;
     }
 
+    // Run temporary tests, remove when proper assembly output tests are in place.
```

Comment:
Prefix with TODO if you know what to do, FIXME if this si know to be broken but ok for now, or XXX if this is wonky but the path forward is not known.